src/ng/interpolate.js

```diff
@@ -44,7 +44,7 @@ $interpolateMinErr.interr = function(text, err) {
       this.label = "This binding is brought you by // interpolation symbols.";
   });
 </script>
-<div ng-app="App" ng-controller="DemoController as demo">
+<div ng-app="customInterpolationApp" ng-controller="DemoController as demo">
```

Comment:
I signed it!

Thanks,
Sakkeer Hussain

On 22 December 2015 at 16:26, Georgios Kalpakas <notifications@github.com>
wrote:

> In src/ng/interpolate.js
> <https://github.com/angular/angular.js/pull/13608#discussion_r48242715>:
>
> > @@ -44,7 +44,7 @@ $interpolateMinErr.interr = function(text, err) {
> >        this.label = "This binding is brought you by // interpolation symbols.";
> >    });
> >  </script>
> > -<div ng-app="App" ng-controller="DemoController as demo">
> > +<div ng-app="customInterpolationApp" ng-controller="DemoController as demo">
>
> I wonder why the protractor test below didn't catch this. Not very
> reassuring... :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/angular.js/pull/13608/files#r48242715>.
>